backend/functions/src/add-contract-to-group.ts

```diff
@@ -90,3 +87,23 @@ export const addcontracttogroup = newEndpoint({}, async (req, auth) => {
 })
 
 const firestore = admin.firestore()
+
+export function canUserAddGroupToMarket(props: {
```

Comment:
Nice job extracting this function! I was just about to comment that you could do this.